ACTIVITY 2/CarbonFootprint.c

```diff
@@ -1,15 +1,96 @@
 #include <stdio.h>
 
-int main(){
-    // 1. Transportation
+/* So i'm going to divide each calculation into different functions to make it cleaner 
+    Emir ur going to do the rest of 4 5 6 similar to what im doing rn
+    Make sure u sent me the rest of the code when ur doing so i can review!
+
+
+*/
+// Part 1 
+float calc_emission_transport() {
     char transportation[20];
-    float transportation_emissions;
-    
-    printf("What is your method of transportation: ");
-    scanf("%3s", &transportation);
+    float emission_factor;
+    float emissions;
+
+    printf("What method of transportation do you use?(Capital first letter) ");
+    scanf("%3s", transportation);
+
+    if ((transportation[0] == 'G' || transportation[0] == 'g') && transportation[1] == 'a' && transportation[2] == 's') {
+        emission_factor = 0.404;
+    } else if ((transportation[0] == 'E' || transportation[0] == 'e') && transportation[1] == 'l' && transportation[2] == 'e') {
+        emission_factor = 0.10425;
+    } else if ((transportation[0] == 'H' || transportation[0] == 'h') && transportation[1] == 'y' && transportation[2] == 'b') {
+        emission_factor = 0.404;
+    } else {
+        emission_factor = 0.0;
+    }
+
+    float miles_per_day;
+
+    do {
+        printf("How many miles do you drive per day? ");
+        scanf("%f", &miles_per_day);
+    } while (miles_per_day < 0);
+
+    emissions = miles_per_day * emission_factor;
+
+    return emissions;
+}
+
+// Part 2
+
+float calc_emission_housing(){
+    float kwh_per_month;
+    float energy_emissions;
+    do {
+        printf("Enter your average monthly electricity consumption (kwh per month): ");
+        scanf("%f", &kwh_per_month);
+    } while (kwh_per_month < 0);

```

Comment:
kwh_per_month is not defined, so I don't know what comparing no-value with 0 would mean.